Source/DafnyDriver/DafnyDriver.cs

```diff
@@ -169,6 +169,23 @@ public enum CommandLineArgumentsResult {
       OK_EXIT_EARLY
     }
 
+    static HashSet<string> SplitOptionValueIntoFiles(HashSet<string> inputs) {
+      var output = new HashSet<string>();
```

Comment:
I would name this `result` instead of `output`. Output sounds like the CLI will print it.